sources/packages/web/src/assets/css/base.scss

```diff
@@ -243,3 +246,26 @@ html * {
 .btn-font-color-light .v-btn__content {
   color: $button-font-color;
 }
+
+// Radio btn style to match with figma
+.input-unset-display-opacity .v-input__control {
```

Comment:
input-unset-display-opacity is this a custom class that will be used in components?

Reply:
yes, the horizontal direction for `v-radio-group` is not working in curent vue 3. I will add a to-do.